build/config.props

```diff
@@ -34,7 +34,7 @@
     <!-- Specifies the SDK version to download to use for testing. The first value represents the channel, the second value represents the exact SDK version to be download. If a version is not specified, the latest version from the channel will be downloaded. 
     Note that multiple SDKs can be downloaded by using `;` as a separator.
     -->
-    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">release/5.0.2xx 5.0.200-servicing.21120.4</CliBranchForTesting>
+    <CliBranchForTesting Condition="'$(CliBranchForTesting)' == ''">release/5.0.2xx 5.0.201</CliBranchForTesting>
   </PropertyGroup>
```

Comment:
Currently `dotnet new template` fails with `5.0.200-servicing.21120.4`

Reply:
Also below command fails so I don't know any good version after this one.
Is there any place where we can find all available versions?
.\dotnet-install.ps1 -Channel release/5.0.2xx -i C:\NuGetProj\NuGet.Client\cli -Version latest -Architecture x64 -NoPath

Reply:
Do I need to open issue somewhere about servicing branch?
![image](https://user-images.githubusercontent.com/8766776/112398083-8ea94700-8cc0-11eb-9101-22f4df55a670.png)

Reply:
@nkolev92 
After we specify latest version in https://github.com/NuGet/NuGet.Client/pull/3924 some `dotnet new template` commands doesn't work anymore.  I tried latest `5.0.202-servicing.21167.13` and problem still persists. Do you have any suggestion?